src/applause-button.js

```diff
@@ -250,10 +255,22 @@ class ApplauseButton extends HTMLElement {
       return;
     }
     const rootColor = this.getAttribute("color") || "green";
-    const style = this._styleRootElement.style;
-    style.fill = rootColor;
-    style.stroke = rootColor;
-    style.color = rootColor;
+    this._styleRootElement.style.fill = rootColor;
+    this._styleRootElement.style.stroke = rootColor;
+    this._styleRootElement.style.color = rootColor;
+    this._clapButton.style.borderColor = rootColor;
+    this._clapButton.style.color = rootColor;
+  }
+
+  _setClapLimitReached() {
+    this._clapButton.setAttribute("aria-disabled", "true");
+    this._clapButton.setAttribute("title", NO_MORE_CLAPS);
+    this._clapButton.setAttribute("aria-description", NO_MORE_CLAPS);
+    this.classList.add("clap-limit-exceeded");
+  }
+
+  _isClapLimitReached() {
+    return this._clapButton.hasAttribute("aria-disabled");
```

Comment:
seems a little non-intuitive to check for the limit this way? Can we not at least check  "clap-limit-exceeded" as before?

Reply:
I would have to respectfully disagree.

`clap-limit-exceeded` is a class, which is concerned with style. Whether or not the button is disabled is a functionality concern: are we allowed to clap, or not?

Note I am deliberately not using `disabled` attribute on the button, for reasons I've given in a past Web COP talk 😉

Reply:
Fair enough - would it make sense to rename the function `_isClapDisabled` instead, as that's really what it's returning?